diff --git a/base/BUILD.gn b/base/BUILD.gn
index 0701aae1a17ee157858bc39db145a3a36ef1631a..5046abebed130aca9c05bb5bf64bd7cba6cd1fa4 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5082,6 +5082,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5756,6 +5757,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
