diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 546ac6a849434f4719fb1b2afd35e91e9f362209..7e6916c76f11cf11c4f966d84dd530b918cc27f9 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -4609,6 +4609,7 @@ source_set("browser") {
   if (!is_android) {
     deps += [ "//chrome/browser/contextual_cueing/internals" ]
   }
+  public_deps += [ "//brave/browser:core" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -4779,6 +4780,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
