diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
index 8a280993ed33a77ab5d3a2d842e57b51407c8dd5..6faaf9bf1f07778cd9254c1739e9856c3466a0d2 100644
--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
+++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list_header.ts
@@ -96,6 +96,7 @@ export class PowerBookmarksListHeaderElement extends CrLitElement {
       label: loadTimeData.getString('sortReverseAlphabetically'),
       lowerLabel: loadTimeData.getString('sortReverseAlphabetically'),
     },
+    { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), },
   ];
   private bookmarksService_: PowerBookmarksService =
       PowerBookmarksService.getInstance();
