diff --git a/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc b/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
index 534c6a2d43d35cd3cf1c2d4eba37a16b4ae3f6ff..af264643d7eeaa62cc52961a3e9bb99c4dffcea4 100644
--- a/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
+++ b/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc
@@ -4472,6 +4472,7 @@ void AddLocalizedStrings(content::WebUIDataSource* html_source,
   policy_indicator::AddLocalizedStrings(html_source);
   AddSecurityKeysStrings(html_source);
 
+  BraveAddLocalizedStrings(html_source, profile);
   html_source->UseStringsJs();
 }
 
