diff --git a/chrome/browser/ui/actions/chrome_action_id.h b/chrome/browser/ui/actions/chrome_action_id.h
index f18b24cf9600092324ae3ee9fefea2b88276035f..51211bb888eb7e96e453958f70abad4afef7be35 100644
--- a/chrome/browser/ui/actions/chrome_action_id.h
+++ b/chrome/browser/ui/actions/chrome_action_id.h
@@ -471,6 +471,9 @@
 // Should be reinserted into CHROME_COMMON_ACTION_IDS when general solution to
 // add action id mappings is implemented.
 #define SIDE_PANEL_ACTION_IDS \
+  E(kActionSidePanelShowPlaylist) \
+  E(kActionSidePanelShowChatUI) \
+  E(kActionSidePanelShowBraveNews) \
   /* Side Panel items */ \
   E(kActionSidePanelShowAboutThisSite) \
   E(kActionSidePanelShowAssistant) \
@@ -498,6 +501,8 @@
     IDC_SHOW_TABS_FROM_OTHER_DEVICES_SIDE_PANEL)
 
 #define TOOLBAR_PINNABLE_ACTION_IDS \
+  E(kActionShowPartitionedStorage) \
+  E(kActionShowPsstIcon) \
   E(kActionHome, IDC_HOME) \
   E(kActionForward, IDC_FORWARD) \
   E(kActionNewIncognitoWindow, IDC_NEW_INCOGNITO_WINDOW) \
