diff --git a/chrome/browser/ui/views/tabs/common/tab_strip_view.cc b/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
index 9f0fc5999548ea79c994f2bcd6c71a3d35f3b2b3..db82d77d7691b5225e9a584399ddfbb3845b07a8 100644
--- a/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
+++ b/chrome/browser/ui/views/tabs/common/tab_strip_view.cc
@@ -208,6 +208,8 @@ void TabStripView::OnChildMoved(TabCollectionNode* moved_node) {
 
   if (is_active_tab || has_focus) {
     switch (moved_node->type()) {
+      case TabCollectionNode::Type::TREE_NODE:
+        break;
       case TabCollectionNode::Type::TAB: {
         const tabs::TabInterface* tab =
             std::get<const tabs::TabInterface*>(moved_node->GetNodeData());
