diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index d5f0ed41e50b0c11b7b7a0de4a1c93a8da40c855..79e020888c9bf14e1eeccb751a3f1479ec082178 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4762,6 +4762,7 @@ const FeatureEntry::FeatureVariation kDictationVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
