diff --git a/chrome/browser/ui/views/location_bar/star_view.cc b/chrome/browser/ui/views/location_bar/star_view.cc
index 890511554c318a0cc8b6001b9b7e21d1d73a2d5f..e3f25549f0aead272ac0e498fd0bf87cb6cadc62 100644
--- a/chrome/browser/ui/views/location_bar/star_view.cc
+++ b/chrome/browser/ui/views/location_bar/star_view.cc
@@ -86,6 +86,7 @@ void StarView::OnWidgetDestroyed(views::Widget* widget) {
 }
 
 void StarView::OnBubbleWidgetChanged(views::Widget* widget) {
+  return;
   CHECK(GetBubble() && widget);
   UpdateTooltipText();
   if (scoped_observation_.IsObserving()) {
@@ -96,7 +97,7 @@ void StarView::OnBubbleWidgetChanged(views::Widget* widget) {
 }
 
 void StarView::UpdateImpl() {
-  bool enabled =
+  bool enabled = false &&
       browser_defaults::bookmarks_enabled && edit_bookmarks_enabled_.GetValue();
   SetVisible(enabled && !delegate()->ShouldHidePageActionIcon(this));
 }
