diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 77b73ded334cb237be71f2a67c8152cce0203660..d07373aa369ed11f4233f4deab8c0c6acdbc2ec9 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -203,6 +203,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  sources += brave_components_update_client_sources deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 if (is_mac) {
