diff --git a/components/feature_engagement/public/feature_list.h b/components/feature_engagement/public/feature_list.h
index 0873211a46fa668ec74fc43d104a09483f79fb39..9902c5748ae34892027bf5d4536009ebfc515247 100644
--- a/components/feature_engagement/public/feature_list.h
+++ b/components/feature_engagement/public/feature_list.h
@@ -620,6 +620,7 @@ DEFINE_VARIATION_PARAM(kIPHiOSTabGroupsDesktopFeature,
 DEFINE_VARIATION_PARAM(kIPHiOSPriceTrackingDesktopFeature,
                        "IPH_iOSPriceTrackingDesktop");
 #endif  // !BUILDFLAG(IS_ANDROID)
+BRAVE_FEATURE_ENGAGEMENT_VARIATION_PARAMS
 
 #if !BUILDFLAG(IS_IOS)
 DEFINE_VARIATION_PARAM(kIPHResumptionRailFeature, "IPH_ResumptionRail");
@@ -636,6 +637,7 @@ DEFINE_VARIATION_PARAM(kIPHSearchPromotionFeature,
 inline constexpr flags_ui::FeatureEntry::FeatureVariation
     kIPHDemoModeChoiceVariations[] = {
 // clang-format off
+        BRAVE_FEATURE_ENGAGEMENT_VARIATION_ENTRIES
 #if BUILDFLAG(IS_ANDROID)
 // keep-sorted start case=no
 // VARIATION_ENTRY_ANDROID_START
