diff --git a/DEPS b/DEPS
index 4f4b57346b3f1254063589f88c71091b1fefdcc8..0ef5696cc8b966bdde8cacee5d08dc038288d2c2 100644
--- a/DEPS
+++ b/DEPS
@@ -4432,7 +4432,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
