diff --git a/third_party/blink/renderer/core/dom/document.cc b/third_party/blink/renderer/core/dom/document.cc
index 894a3e46b2cfac21526d4db47def8c1b0a3205c1..9309f54d5995e49a175be48624380f0db826c117 100644
--- a/third_party/blink/renderer/core/dom/document.cc
+++ b/third_party/blink/renderer/core/dom/document.cc
@@ -7601,6 +7601,7 @@ KURL Document::CompleteURLWithOverride(
   if (result.IsEmpty()) {
     result = Encoding().IsValid() ? KURL(base_url_override, url, Encoding())
                                   : KURL(base_url_override, url);
+    result = RewriteBraveToChrome(result);
     url_cache_.Put(base_url_override, url, result);
   }
   // If the conditions are met for
